homeassistant/components/co2signal/sensor.py

```diff
@@ -114,6 +114,11 @@ def __init__(
             f"{coordinator.entry_id}_{description.unique_id or description.key}"
         )
 
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self._description.key in self.coordinator.data["data"]
+
     @property
     def state(self) -> StateType:
         """Return sensor state."""
```

Comment:
Will we get a `KeyError` here?
Or is `state` never called when `available` is `False`?

Reply:
State is not called when not available.